build/sync.js

```diff
@@ -0,0 +1,42 @@
+const path = require('path');
+const fs = require('fs');
+const chokidar = require('chokidar');
+
+const shared = path.join(__dirname, '..', 'shared', 'src');
+const frontend = path.join(__dirname, '..', 'frontend', 'src', 'shared');
+const backend = path.join(__dirname, '..', 'backend', 'src', 'shared');
+
+const log = (message) => {
+  console.log(`[${new Date().toLocaleTimeString()}]`, message);
+}
+
+/**
+ * Stolen from https://stackoverflow.com/a/22185855
```

Comment:
This is not good, as we had to license this in order to integrate it here and stack overflows license is in particual copyleft and not of any fit for this project. Remove this code and rewrite it yourself.  

In detail:  
The post on stackoverflow was on `Mar 5 '14 at 0:04`, so this is licensed to us under CC BY-SA 3.0 license according to https://stackoverflow.com/help/licensing.  
To redistribute this, we had to  
> ShareAlike — If you remix, transform, or build upon the material, you must distribute your contributions under the same license as the original.   

According to https://creativecommons.org/licenses/by-sa/3.0/ which is incompatible with the license (MIT) of our project.